unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/SerializableKeyValuePair.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+
+[Serializable]
+public class SerializableKeyValuePair<TKey, TValue>
```

Comment:
As it is a more generic class, maybe we should move it to another more common place. Wdyt?